drivers/openxr/openxr_device.cpp

```diff
@@ -1655,17 +1660,19 @@ void OpenXRDevice::pre_render() {
 #endif
 	}
 
-	if (frame_state.shouldRender) {
-		// let's start our frame..
-		XrFrameBeginInfo frame_begin_info = {
-			XR_TYPE_FRAME_BEGIN_INFO, // type
-			nullptr // next
-		};
-		result = xrBeginFrame(session, &frame_begin_info);
-		if (XR_FAILED(result)) {
-			print_line("OpenXR: failed to being frame [", get_error_string(result), "]");
-			return;
-		}
+	if (!frame_state.shouldRender) {
```

Comment:
ok, I probably misunderstood Rune here, I got the impression xrBeginFrame was only needed if we ended up rendering but it does make more sense. 

I'll remove this top if statement, with the logic now deviced between pre_render and pre_draw_viewport, that last one returning false will prevent Godot from rendering a frame.